tools/iframe-test-resizer.html

```diff
@@ -0,0 +1,36 @@
+<!-- this file can be loaded directly from a browser and used to test embedding groups and projects within iframes
+see https://github.com/DemocracyLab/CivicTechExchange/issues/978 
+change groups/inframe to projects/inframe below to test embedding a project
+This if for testing with iframe resizer
+ -->
+<!DOCTYPE html>
+<html lang="en" style="height: 100%">
+  <head>
+    <meta charset="utf-8" />
+    <title>iframe test</title>
+  </head>
+  <body>
+    <div style="width: 100vw;">
+        <h1>This is a landing page with Group projects:</h1>
+        <style>
+          iframe {
+            width: 1px;
+            min-width: 100%;
```

Comment:
100vw on the parent and 100% min-width here breaks mousewheel scroll - it forces the mouse into the iframe with no horizontal room to escape because the frame is filling all available width. Changing this to  `min-width: calc(100% - 30px);` so there's room for the mouse to move outside the frame would probably work. Removing the 100vw off the parent div may also be enough.

Reply:
I made the width change to the parent div. The min-width 100% is a recommendation from the iframe-resizer.

Reply:
Sounds good, thanks. It was the combination of both -- so changing either is fine. If the resizer suggests min-width: 100% on the iframe, then let's keep that by all means.